pkg/tms/tmsUtils.go

```diff
@@ -105,7 +110,7 @@ type CommunicationInterface interface {
 }
 
 type Options struct {
-	TmsServiceKey            string                 `json:"tmsServiceKey,omitempty"`
+	ServiceKey               string                 `json:"tmsServiceKey,omitempty"`
```

Comment:
One question here:
```golang
ServiceKey    string    `json:"tmsServiceKey,omitempty"`
```
The json key `tmsServiceKey` is not actually used as far as I can see. Should we leave it as `tmsServiceKey` or change it to `serviceKey`, in case we someday want to unmarshall this from a json string? Because if we ever unmarshall this from a json string we are likely to oversee this.

Another option is to remove all these json key fields from the struct, as they are probably a leftover from when only the `tmsUpload` step existed.

Your thought @artembannikov @gerstneralex ?